src/Symfony/Component/Notifier/Tests/Exception/MultipleExclusiveOptionsUsedExceptionTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Tests\Exception;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Exception\MultipleExclusiveOptionsUsedException;
+
+final class MultipleExclusiveOptionsUsedExceptionTest extends TestCase
```

Comment:
I'm removing `final` : a test case is not an API, we don't need extra visual debt (same as `void`) ;)